src/js/lib/monster.util.js

```diff
@@ -1245,6 +1234,48 @@ define(function(require) {
 			);
 	}
 
+	/**
+	 * Generates a string of `length` random characters chosen from either a
+	 * preset or a custom string of characters.
+	 * @param  {Number} length  Number of characters to include.
+	 * @param  {String} pPreset Characters to choose from.
+	 * @return {String}         A string of random characters.
+	 */
+	function randomString(length, pPreset) {
+		if (!_.isNumber(length)) {
+			throw new TypeError('Expected a number for `length`');
+		}
+		if (!_.isUndefined(pPreset) && !_.isString(pPreset)) {
+			throw new TypeError('Expected a string for `preset`');
+		}
+		var string = _.isUndefined(pPreset)
+			? 'safe'
+			: pPreset;
+		var presets = {
+			alpha: '1234567890abcdefghijklmnopqrstuvwxyz',
+			letters: 'abcdefghijklmnopqrstuvwxyz',
+			numerals: '1234567890',
+			hex: '1234567890abcdef',
+			safe: '23456789abcdefghjkmnpqrstuvwxyz'
+		};
+		var preset = _
+			.chain(presets)
+			.get(string, string)
+			.shuffle()
+			.value();
+		var getRandomItem = function() {
```

Comment:
There could be a single `var` for all the variables declaration (a catch from @masmerino13)